node/cmd/guardiand/admintemplate.go

```diff
@@ -204,6 +207,13 @@ func init() {
 	AdminClientAccountantModifyBalanceCmd.Flags().AddFlagSet(moduleFlagSet)
 	TemplateCmd.AddCommand(AdminClientAccountantModifyBalanceCmd)
 
+	// flags for the core-bridge-set-message-fee command
+	coreBridgeSetMessageFeeFlagSet := pflag.NewFlagSet("core-bridge-set-message-fee", pflag.ExitOnError)
+	coreBridgeSetMessageFeeChainId = coreBridgeSetMessageFeeFlagSet.String("chain-id", "", "Chain ID")
+	coreBridgeSetMessageFeeMessageFee = coreBridgeSetMessageFeeFlagSet.String("message-fee", "", "New message fee")
```

Comment:
Same here, is it possible to use a uint64 value instead of a string?

Reply:
Done!